src/rules/if_change_then_change.rs

```diff
@@ -97,11 +97,11 @@ pub fn ictc(hunks: &Vec<Hunk>) -> anyhow::Result<Vec<diagnostic::Diagnostic>> {
             severity: diagnostic::Severity::Error,
             code: "if-change-then-change".to_string(),
             message: format!(
-                "{}[{}, {}) was modified, but no if-change-then-change block in {} was modified",
+                "Expected change in {} because {}[{}, {}) was modified",
```

Comment:
Did you intend for this mismatch
[)

Should it be
"Expected change in {} because {}[{}, {}] was modified",